GDAP Documentation (beta APIs) (#16193)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/delegatedadminaccessassignment.md

---
title: "delegatedAdminAccessAssignment resource type"

> description: "Represents an assignment of administrative roles to a Microsoft partner security group."

Reviewer: ```suggestion
description: "Represents an assignment of administrative roles to a Microsoft partner access container."
```

Reviewer 2: +1

> description: "Represents an assignment of administrative roles to a Microsoft partner security group."

Reviewer 2: Minor fix @adtangir 
```suggestion
description: "Represents an assignment of administrative roles to a Microsoft partner's security group."
```

## Methods
|Method|Return type|Description|
|:---|:---|:---|
|[Create delegatedAdminAccessAssignment](../api/delegatedadminrelationship-post-accessassignments.md)|[delegatedAdminAccessAssignment](delegatedadminaccessassignment.md)|Create a new **delegatedAdminAccessAssignment** object.|
|[List delegatedAdminAccessAssignments](../api/delegatedadminaccessassignment-list.md)|[delegatedAdminAccessAssignment](delegatedadminaccessassignment.md) collection|Get a list of the **delegatedAdminAccessAssignment** objects and their properties.|
|[Get delegatedAdminAccessAssignment](../api/delegatedadminaccessassignment-get.md)|[delegatedAdminAccessAssignment](delegatedadminaccessassignment.md)|Read the properties and relationships of a **delegatedAdminAccessAssignment** object.|
|[Update delegatedAdminAccessAssignment](../api/delegatedadminaccessassignment-update.md)|[delegatedAdminAccessAssignment](delegatedadminaccessassignment.md)|Update the properties of a **delegatedAdminAccessAssignment** object.|

> |[Delete delegatedAdminAccessAssignment](../api/delegatedadminaccessassignment-delete.md)|None|Deletes a **delegatedAdminAccessAssignment** object.|

Reviewer 3: ```suggestion
|[Delete delegatedAdminAccessAssignment](../api/delegatedadminaccessassignment-delete.md)|None|Delete a **delegatedAdminAccessAssignment** object.|
```

## Properties
|Property|Type|Description|
|:---|:---|:---|
|accessContainer|[delegatedAdminAccessContainer](../resources/delegatedadminaccesscontainer.md)|The access container through which members are assigned access. For example, a security group.|
|accessDetails|[delegatedAdminAccessDetails](../resources/delegatedadminaccessdetails.md)|The access details containing the identifiers of the administrative roles that the partner is assigned in the customer tenant.|
|createdDateTime|DateTimeOffset|The date and time in ISO 8601 format and in UTC time when the access assignment was created. Read-only.|
|id|String|The unique identifier of the access assignment. Read-only. Inherited from [entity](../resources/entity.md).|
|lastModifiedDateTime|DateTimeOffset|The date and time in ISO 8601 and in UTC time when this access assignment was last modified. Read-only.|

> |status|delegatedAdminAccessAssignmentStatus|The status of the access assignment. The possible values are: `pending`, `active`, `deleting`, `deleted`, `error`, `unknownFutureValue`.|

Reviewer 3: ```suggestion
|status|delegatedAdminAccessAssignmentStatus|The status of the access assignment. Read-only. The possible values are: `pending`, `active`, `deleting`, `deleted`, `error`, `unknownFutureValue`.|
```
